### PR TITLE
Compulsory Savings no longer required

### PR DESCRIPTION
**Pull request by CRMTravis**

### CODE REVIEW

File: force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponentController.js [hunk 1]
```diff
@@ -187,9 +187,6 @@
       "otherReason",
       "indicateRepaymentMethod",
       "dedicatedMonthlyRepaymentDate",
-      "proposedSavings",
```
Comment (RemarioCRM) [inline force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponentController.js:190]: did you test this by saving?

File: force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponent.cmp [hunk 2]
```diff
@@ -582,8 +582,7 @@
               label="Amount(JMD)"
               value="{!v.ChildContainer.proposedSavingsAmount}"
               formatter="currency"
-              required="true"
-              min="1000"
+              required="false"
```
Comment (RemarioCRM) [inline force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponent.cmp:585]: no more min ?

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponentController.js (hunk 1 above)
Comment (CRMTravis) [reply force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponentController.js:190]: This was tested yesterday Remario

File: force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponent.cmp (hunk 2 above)
Comment (CRMTravis) [reply force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponent.cmp:585]: Validation was to be removed.

File: force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponentController.js (hunk 1 above)
Comment (RemarioCRM) [reply force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponentController.js:190]: you made this specific change today right?

Comment (CRMTravis) [reply force-app/main/default/aura/AutoLoanFeaturesLineItemComponent/AutoLoanFeaturesLineItemComponentController.js:190]: No sir

**Review by RemarioCRM — APPROVED** [review]

**Pull request merged**